easy-install.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python3
+import argparse
+import subprocess
+import os
+import sys
+import time
+from shutil import which
+import platform
+from secrets import token_bytes
+from base64 import b64encode
+
+CRED = "\033[31m"
+CEND = "\033[0m"
+CGRN = "\33[92m"
+CYLW = "\33[93m"
+
+
+def clone_frappe_docker_repo() -> None:
+    try:
+        subprocess.run(
+            ["git", "clone", "https://github.com/frappe/frappe_docker"], check=True
```

Comment:
This assumes git is installed, which by default isn't on most distros.

Reply:
This should be either part of instruction OR handled by script.

Alternatively, download the zip over HTTP and expand. We don't really need a git repository.

Reply:
> From "broad goal" POV we should also limit ourselves to few Linux distros. Ubutnu/Debian dervatives at first and maybe Fedora someday later. We can assume that other crazy distro users can setup frappe_docker themselves.

Once we have docker installed, distro doesn't matter.
If docker is not installed, only ubuntu/debian/fedora/centos/rhel or distros supported by `curl -fsSL get.docker.com | bash` will install docker. Any unsupported distro by the script shows message like:

```
ERROR: Unsupported distribution 'alpine'
```

more about script here https://github.com/docker/docker-install

In case of unsupported distro, if someone manages to install `docker` and `docker compose` then the script will proceed.